src/node-explorer-provider.ts

```diff
@@ -777,29 +781,29 @@ export class FileExplorer extends vscode.TreeItem {
     public readonly uri: vscode.Uri,
     public readonly type: vscode.FileType,
     public readonly context?: string,
-    public readonly collapsibleState: vscode.TreeItemCollapsibleState = type ===
+    public readonly collapsibleState: vscode.TreeItemCollapsibleState = type &
```

Comment:
Any reason we're changing to `&` everywhere instead of basic equality?

Reply:
As I said above, a symbolic link to a file would now be returned as `vscode.FileType.SymbolicLink | vscode.FileType.File`, so it wouldn't compare equal anymore. `&` lets us check that the same bit is set in the `vscode.FileType` bitfield: https://vscode-api.js.org/enums/vscode.FileType.html